scripts/release_scripts/update_configs.py

```diff
@@ -171,6 +172,10 @@ def add_redishost():
     """Adds redishost key to feconf.py."""
     redishost = getpass.getpass(
         prompt=('Enter REDISHOST from the release process doc.'))
+    redishost = redishost.strip()
+
+    if re.match(r'^\d{1,3}\.\d{1,3}\.\d{1,3}\.\d{1,3}$', redishost) is None:
+        raise Exception('Invalid REDISHOST.')
```

Comment:
Perhaps just ask them to try again. The user may have typed an extra thing by accident. Make sure the retry prompt explicitly requests a retry and asks the user to enter a valid IP address.

Reply:
Done.